tracing/providers/opentelemetry/src/main/java/io/helidon/tracing/providers/opentelemetry/HelidonOpenTelemetry.java

```diff
@@ -91,6 +92,19 @@ public static io.helidon.tracing.Span create(Span span, Baggage baggage) {
         return new OpenTelemetrySpan(span, baggage, SPAN_LISTENERS.get());
     }
 
+    /**
+     * Wrap an open telemetry span builder.
+     *
+     * @param spanBuilder open telemetry span builder
+     * @param helidonTracer Helidon {@link io.helidon.tracing.Tracer} to use in creating the wrapping span builder
+     * @return Helidon {@link io.helidon.tracing.Span.Builder}
+     */
+    public static  io.helidon.tracing.Span.Builder<?> create(SpanBuilder spanBuilder,
```

Comment:
there are two spaces after `static` instead of one

Reply:
Fixed.